src/main/java/org/waarp/openr66/commander/Commander.java

```diff
@@ -508,13 +521,6 @@ public void run() {
                 }
                 logger.error("Database SQL Error: Cannot execute Commander", e);
                 return;
-            } catch (WaarpDatabaseException e) {
```

Comment:
Removing this catch entirely means that `WaarpDatabaseException`s that might be thrown elsewhere in the block (l. 461 & 462) won't be caught. 

Why not only catch `WaarpDatabaseNoDataException`s in the new block and let this one to handle all the others `waarpDatabaseException`s (the processing is the same in the new block (i.e. `return`) ?